library/kani/src/lib.rs

```diff
@@ -14,6 +20,15 @@ pub use futures::block_on;
 #[allow(deprecated)]
 pub use invariant::Invariant;
 
+#[cfg(feature = "exe_trace")]
```

Comment:
Done. I moved both `exe_trace_run` and the exe trace implementation of `kani::any_raw_internal` to a separate module. I chose to use a `#[cfg(feature)]` inside the `kani::any_raw_internal` implementation rather than on the whole function itself since I feel like its confusing to have two different functions that both map to the same function names under difference features.